### ISSUE TITLE
Sketch File Input Creates Empty components.js Output

### ISSUE THREAD

**Issue by pburtchaell** [souporserious/unicon#7]
Hello, I'm super interested in using this package! However, when I run the CLI with a Sketch file, the output is an empty `components.js` file.

Here's my package.json:

```
{
    "name": "icons",
    "version": "0.0.1",
    "scripts": {
        "start": "unicon sketch ./sketch/ios.sketch --transformer json"
    },
    "devDependencies": {
        "unicon-cli": "0.0.3",
        "unicon-transformer-json": "0.0.1",
        "unicorn": "0.0.1"
    },
    "dependencies": {}
}
```

My Sketch file contains three artboards, each with a primitive shape (rect, circle, triangle).

**Comment by souporserious** [comment]
Glad to here you're interested in using unicon! 🙂

Weird that it's producing an empty file 🤔. Is it possible you can share your Sketch file? It should be picking up those artboards. Also, what version of Sketch are you using?

**Comment by pburtchaell** [comment]
Yeah, it is strange. I'm on Sketch 51.2.

Here's the file: https://www.dropbox.com/s/j6zifm4lbbji76r/shapes.sketch?dl=0

**Comment by souporserious** [comment]
Thanks for the test file! 🙏It seems to be working ok on my end. I'm wondering if the SketchTool is silently failing around [this line](https://github.com/souporserious/unicon/blob/master/packages/unicon/src/get-svgs-from-sketch.js#L9). I'll need add some better logging to try and diagnose what's going on. Does your Sketch.app live in your Applications folder?

**Comment by pburtchaell** [comment]
Yeah, I checked and my Sketch file is also at the same path. I also checked package contents to make sure I had `sketchtool/bin/sketchtool`. 

Let me see if I can dig into this more and create a test repo.

**Comment by souporserious** [comment]
Awesome, thanks! I'll try and do some more digging when I get a chance. I want to add some way to generate a debug file or something to make things like this hopefully easier to figure out 😅.
